Xamarin.PropertyEditing.Mac/Controls/Custom/CommandMenuItem.cs

```diff
@@ -0,0 +1,59 @@
+﻿using System;
+using System.Windows.Input;
+using AppKit;
+
+namespace Xamarin.PropertyEditing.Mac
+{
+	public class CommandMenuItem : NSMenuItem
+	{
+		ICommand command;
+
+		public ICommand Command {
+			get { return command; }
+			set {
+				if (this.command != null)
+					this.command.CanExecuteChanged -= CanExecuteChanged;
+
+				this.command = value;
+				this.command.CanExecuteChanged += CanExecuteChanged;
+			}
+		}
+
+		public CommandMenuItem (string title) : base (title)
+		{
+			HookUpCommandEvents ();
+		}
+
+		public CommandMenuItem (string title, ICommand command) : this (title)
+		{
+			this.Command = command;
+
+			HookUpCommandEvents ();
```

Comment:
You already did that by calling `this(title)`

Reply:
Nicely spotted. I must have thought I was still calling base there.